src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -332,13 +335,71 @@ public override IntermediateNode Visit(TexlCallNode node, IRTranslatorContext co
                     }
                 }
 
+                args = ReplaceBlankWithDefault(args, func);
```

Comment:
I'd add a code comment differentiating these kinds of coercions (blank handling, etc.) from the logic in MaybeInjectCoercion, unless they should really be merged

Reply:
updated.